internal/benchrunner/runners/pipeline/benchmark.go

```diff
@@ -0,0 +1,292 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipeline
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/benchrunner"
+	"github.com/elastic/elastic-package/internal/elasticsearch/ingest"
+)
+
+const (
+	// How many top processors to return.
+	numTopProcs = 10
```

Comment:
If we expose it through the cli, it will use the value for all the datastreams in the package, in some cases based on the size of the events used to test, this might cause some simulations to fail with a 413. This means some packages might require different num_doc values for each datastream, and this is why I added it to the benchmark config instead. Plus if the intention is to use it in CI, it is much less flexible to use because the same reason. WDYT?